pulse2percept/models/granley2021.py

```diff
@@ -0,0 +1,559 @@
+"""`BiphasicAxonMapModel`"""
+from attr import Attribute
+import numpy as np
+from copy import deepcopy
+import sys
+
+from numpy.core.fromnumeric import size
+
+from . import AxonMapModel, AxonMapSpatial, TemporalModel, Model
+from ..implants import ProsthesisSystem, ElectrodeArray
+from ..stimuli import BiphasicPulseTrain, Stimulus
+from ..datasets import load_nanduri2012
+from ..percepts import Percept
+from ..utils import FreezeError
+
+from ._granley2021 import fast_biphasic_axon_map
+from .base import NotBuiltError
+
+
+class DefaultBrightModel():
+    """
+    Default model to be used for brightness scaling in BiphasicAxonMapModel
+    Implements Eq 4 from [Granley2021]_ 
+    Fit using data from [Nanduri2012]_ and [Weitz2015]_
+
+    Parameters:
+    ------------
+    do_thresholding : bool, optional
+        Set to true to enable probabilistic phosphene appearance at near-threshold 
+        amplitudes
+    **kwargs : dict, optional 
```

Comment:
In all of these models, it would be tremendously useful to spell out the meaning of the different A0-9 parameters, rather than `**kwargs`